core/tracing/hooks.go

```diff
@@ -148,8 +148,8 @@ type (
 	LogHook = func(log *types.Log)
 
 	CaptureArbitrumTransferHook   = func(from, to *common.Address, value *big.Int, before bool, purpose string)
-	CaptureArbitrumStorageGetHook = func(key common.Hash, depth int, before bool)
-	CaptureArbitrumStorageSetHook = func(key, value common.Hash, depth int, before bool)
+	CaptureArbitrumStorageGetHook = func(addr common.Address, key, mappedKey common.Hash, depth int, before bool)
+	CaptureArbitrumStorageSetHook = func(addr common.Address, key, mappedKey, value common.Hash, depth int, before bool)
```

Comment:
addr should always be the arbosStorage address. We can add it here but not really necessary.
We shouldn't need mapped key because that's what "key" should mean. Storage is just key-value pairs and we don't care about the non-mapped key.